misc/dbt-materialize/CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # dbt-materialize Changelog
 
-## 1.0.3 - 2021-02-27
+## 1.0.4 - 2022-03-16
```

Comment:
Ah so this needs to match the dbt-postgres version by convention, so this new release should be 1.0.3.post1!

Reply:
Thank you! I wondered.